apps/discovery_streams/LICENSE

```diff
@@ -1,201 +0,0 @@
-                                 Apache License
```

Comment:
Do we not need a license for the individual apps since it's top level? I guess that just makes sense.

Reply:
I didn't see them for the others? Maybe A&C this?